Project/Versie_3/style.css

```
@@ -36,11 +36,11 @@ body {
   display: -ms-flexbox;
   display: flex;
   -webkit-box-pack: justify;
-      -ms-flex-pack: justify;
-          justify-content: space-between;
+  -ms-flex-pack: justify;
```

just adding a comment
```suggestion
  -ms-flex-pack: justify;
```

---

resolved as this is just a comment. No spamming please..